engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -106,6 +110,15 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
 
   private val calls = workflowDescriptor.backendDescriptor.workflowNamespace.workflow.calls
 
+  // TODO: We should probably create a trait which loads all the configuration (once per application), and let classes mix it in
+  // to avoid doing ConfigFactory.load() at multiple places
+  val MaxRetries = Try(ConfigFactory.load().getInt("system.max-retries")) match {
```

Comment:
@cjllanwarne Something like this? I didn't want to jam this value in CromwellBackend or BackendConfiguration since it doesn't quite belong there and I'm trying to avoid more singleton accesses in this.